src/test/java/api/support/fixtures/LoansFixture.java

```diff
@@ -138,8 +138,17 @@ public Response declareItemLost(UUID loanId,
 
     JsonObject request = builder.create();
 
-    return restAssuredClient.post(request,
-      declareLoanItemLostURL(loanId.toString()), "declare-item-lost-request");
+    return restAssuredClient.post(request, declareLoanItemLostURL(loanId.toString()),
+      204, "declare-item-lost-request");
+  }
+
+  public Response attemptDeclareItemLost(
+    UUID loanId, DeclareItemLostRequestBuilder builder) {
+
+    JsonObject request = builder.create();
+
+    return restAssuredClient.post(request, declareLoanItemLostURL(loanId.toString()),
+      "attempt-declare-item-lost-request");
   }
```

Comment:
Similarly to comments in other pull requests, I think it makes sense to start splitting out aspects of the loan lifecycle into separate fixtures, due to it's increasing size.

I think declare lost might one of those aspects. What do you think?

Reply:
Moved to `DeclareLostFixtures`